src/Features/CSharp/Portable/CodeRefactorings/SyncNamespace/CSharpChangeNamespaceService.cs

```diff
@@ -194,6 +201,23 @@ protected override SyntaxList<MemberDeclarationSyntax> GetMemberDeclarationsInCo
             // The name will be resolved by adding proper import.
             old = @new = nameRef;
             return false;
+
+#nullable enable
```

Comment:
Just curious... Is this the general recommended way to enable nullable in the IDE layer? As in, use if for newly added methods even if the whole file is not nullable enabled?

Reply:
I would say the recommendation is to "nullable enable as much as you can get away with", but sometimes that's too big of an undertaking at once. I've totally done this in the VisualStudioWorkspace type because there's just a lot of code there, but I don't want to not annotate new code going forward.

Reply:
That said, I'm not sure why we wouldn't do this for the entire method rather than just the local method inside of it....